criteria.lua

```diff
@@ -53,6 +53,7 @@ local demo_steps = {
 -- ---------------------------------------------------------------------------------------------------------------------
 local function create_enemy_forces_bar(step_index)
   if enemy_forces_bar then
+    enemy_forces_bar:SetPoint("TOPLEFT", step_frames[step_index - 1], "BOTTOMLEFT", 0, -5)
```

Comment:
Update point for `enemy_forces_bar` even if we take previously created `enemy_forces_bar` for reuse. Should fix the problem. But can not test it because of WoW lagging right now.

Reply:
Ok no problem, i will continue testing

Reply:
Recently finish Atal'Dazar and then Waycrest Manor - everything fine.